### PR TITLE
Added: Kotlin Math abs function✨

### LINKED ISSUE #3511: [Term Entry] Kotlin Math .abs()
### Reason/inspiration (optional)

We would like a new term entry on the `.abs()` method for Kotlin: [Math](https://www.codecademy.com/resources/docs/kotlin/math-methods). The entry should go in a new file under `docs/content/kotlin/concepts/math-methods/terms/abs/abs.md`.

The entry should describe what the method does. Be sure to include the following sections:

- A "Syntax" section with a ` ```pseudo` code snippet.
- An "Example" section with a ` ```kotlin `code snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by VividhPandey003**
### Description

Added Kotlin Math abs function documentation 

Closes #3511 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/kotlin/concepts/math-methods/terms/abs/abs.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    val absoluteY = abs(y)
+    val absoluteZ = abs(z)
+
+    println("Absolute value of x: $absoluteX")
+    println("Absolute value of y: $absoluteY")
+    println("Absolute value of z: $absoluteZ")
+}
+```
+
+In this example, we import the abs function from the `kotlin.math` library and use it to calculate the absolute values of various numeric expressions. The results are then printed to the console.
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/math-methods/terms/abs/abs.md:49]:
```suggestion
In this example, the **`abs`** function is imported from the `kotlin.math` library and employed to calculate the absolute values of different numeric expressions. The results are subsequently printed to the console. 
```
Refrain from using pronouns

File: content/kotlin/concepts/math-methods/terms/abs/abs.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+The output of this code will be:
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/math-methods/terms/abs/abs.md:51]:
```suggestion
The output of this code will be:

```
adding a line after this

File: content/kotlin/concepts/math-methods/terms/abs/abs.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+```shell
+Absolute value of x: 6.5
+Absolute value of y: 2
+Absolute value of z: 340.8
+```
+
+The .abs() method is particularly useful when you need to ensure that a value is treated as positive, regardless of its original sign.
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/math-methods/terms/abs/abs.md:58]:
```suggestion
The **`.abs()`** method is notably useful when ensuring that a value is treated as positive, irrespective of its original sign.
```
Refrain from using pronouns.

File: content/kotlin/concepts/math-methods/terms/abs/abs.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+```pseudo
+Math.abs(n)
+```
+
+`n`: A numeric expression (e.g., `Int`, `Long`, `Double`, `Float`, etc.) for which you want to calculate the absolute value.
+
+The method takes the numeric expression n as its argument and returns the absolute value, ensuring that the result is always a non-negative value.
+
+### Example
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/math-methods/terms/abs/abs.md:29]:
```suggestion
## Example 
```

File: content/kotlin/concepts/math-methods/terms/abs/abs.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+## Description
+
+The `.abs()` method in Kotlin's Math library is used to calculate the absolute value of a numeric expression. The absolute value is the non-negative value of a number, effectively removing its negative sign, if present.
+
+### Syntax
+
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/math-methods/terms/abs/abs.md:19]:
```suggestion
```

File: content/kotlin/concepts/math-methods/terms/abs/abs.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+## Description
+
+The `.abs()` method in Kotlin's Math library is used to calculate the absolute value of a numeric expression. The absolute value is the non-negative value of a number, effectively removing its negative sign, if present.
+
+### Syntax
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/math-methods/terms/abs/abs.md:18]:
```suggestion
## Syntax
```

File: content/kotlin/concepts/math-methods/terms/abs/abs.md (hunk 3 above)
Comment (dakshdeepHERE) [inline content/kotlin/concepts/math-methods/terms/abs/abs.md:58]: You can add a working codebyte after this. Try to keep the code different from above example.

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
I have requested few changes. Please commit them from your side. 🚀

File: content/kotlin/concepts/math-methods/terms/abs/abs.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+
+
+```pseudo
+Math.abs(n)
+```
+
+`n`: A numeric expression (e.g., `Int`, `Long`, `Double`, `Float`, etc.) for which you want to calculate the absolute value.
+
+The method takes the numeric expression n as its argument and returns the absolute value, ensuring that the result is always a non-negative value.
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/math-methods/terms/abs/abs.md:27]:
```suggestion
`n`: A numeric expression (e.g., `Int`, `Long`, `Double`, `Float`, etc.) for which you want to calculate the absolute value.

The method takes the numeric expression n as its argument and returns the absolute value, ensuring that the result is always a non-negative value.
```
```suggestion
`n`: A numeric expression (e.g., `Int`, `Long`, `Double`, `Float`, etc.) for which you want to calculate the absolute value.

The method takes the numeric expression n as its argument and returns the absolute value, ensuring that the result is always a non-negative value.
```
```suggestion
`n`: A numeric expression (e.g., `Int`, `Long`, `Double`, `Float`, etc.) for which you want to calculate the absolute value. The method takes the numeric expression n as its argument and returns the absolute value, ensuring that the result is always a non-negative value.
```
There is no need to leave space here.

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]

**Comment by VividhPandey003** [comment]
@dakshdeepHERE Committed the suggested changes. Would you please review them once.

File: content/kotlin/concepts/math-methods/terms/abs/abs.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+
+The output of this code will be:
+
+```shell
+Absolute value of x: 6.5
+Absolute value of y: 2
+Absolute value of z: 340.8
+```
+
+The **`.abs()`** method is notably useful when ensuring that a value is treated as positive, irrespective of its original sign.
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/math-methods/terms/abs/abs.md:56]:
```suggestion
> **Note**: The `.abs()` method is notably useful when ensuring that a value is treated as positive, irrespective of its original sign.
```

File: content/kotlin/concepts/math-methods/terms/abs/abs.md [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+    val absoluteY = abs(y)
+    val absoluteZ = abs(z)
+
+    println("Absolute value of x: $absoluteX")
+    println("Absolute value of y: $absoluteY")
+    println("Absolute value of z: $absoluteZ")
+}
+```
+
+In this example, the **`abs`** function is imported from the `kotlin.math` library and employed to calculate the absolute values of different numeric expressions. The results are subsequently printed to the console.
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/math-methods/terms/abs/abs.md:46]:
```suggestion
In this example, the `abs` function is imported from the `kotlin.math` library and employed to calculate the absolute values of different numeric expressions. The results are subsequently printed to the console. 
```

File: content/kotlin/concepts/math-methods/terms/abs/abs.md [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+  - 'Methods'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+## Description
+
+The `.abs()` method in Kotlin's Math library is used to calculate the absolute value of a numeric expression. The absolute value is the non-negative value of a number, effectively removing its negative sign, if present.
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/math-methods/terms/abs/abs.md:16]:
```suggestion
The **`.abs()`** method in Kotlin's Math library is used to calculate the absolute value of a numeric expression. The absolute value is the non-negative value of a number, effectively removing its negative sign, if present.
```

File: content/kotlin/concepts/math-methods/terms/abs/abs.md [hunk 11]
```diff
@@ -0,0 +1,56 @@
+---
+Title: '.abs()'
+Description: 'Returns the absolute value of the argument.'
+Subjects:
+  - 'Computer Science'
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/math-methods/terms/abs/abs.md:5]:
```suggestion
  - 'Computer Science'
  - 'Code Foundations'
```

File: content/kotlin/concepts/math-methods/terms/abs/abs.md [hunk 12]
```diff
@@ -0,0 +1,56 @@
...
+    val z = -340.8
+
+    val absoluteX = abs(x)
+    val absoluteY = abs(y)
+    val absoluteZ = abs(z)
+
+    println("Absolute value of x: $absoluteX")
+    println("Absolute value of y: $absoluteY")
+    println("Absolute value of z: $absoluteZ")
+}
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/math-methods/terms/abs/abs.md:43]:
```suggestion
import kotlin.math.abs

fun main() {
  val x = -6.5
  val y = 2
  val z = -340.8

  val absoluteX = abs(x)
  val absoluteY = abs(y)
  val absoluteZ = abs(z)
  println("Absolute value of x: $absoluteX")
  println("Absolute value of y: $absoluteY")
  println("Absolute value of z: $absoluteZ")
}
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
Thank you for the commit here are few more. Commit these so we can move forward to 2nd review. :)

**Comment by VividhPandey003** [comment]
> Thank you for the commit here are few more. Commit these so we can move forward to 2nd review. :)

Done

**Comment by dakshdeepHERE** [comment]
@VividhPandey003 LGTM 🚀

**Review by dakshdeepHERE — APPROVED** [review]

**Review by caupolicandiaz — APPROVED** [review]
@VividhPandey003 I made some direct edits to update the formatting. Looks good to merge

**Pull request merged**